remap-keys.lisp

```diff
@@ -25,10 +25,13 @@
 
 (in-package #:stumpwm)
 
-(export '(define-remapped-keys))
-
+(export '(define-remapped-keys *remap-keys-enable*))
+ 
 (defvar *remap-keys-window-match-list* nil)
 
+(defvar *remap-keys-enable* t
```

Comment:
Could you rename this to `*remapped-keys-enabled-p*` so it aligns with `*hooks-enabled-p*`?

Reply:
ok